libr/core/cmd_search.c

```diff
@@ -1625,7 +1625,7 @@ static void do_esil_search(RCore *core, struct search_parameters *param, const c
 		}
 		/* hook addrinfo */
 		core->anal->esil->cb.user = core;
-		r_anal_esil_set_op (core->anal->esil, "AddrInfo", esil_addrinfo);
+		r_anal_esil_set_op (core->anal->esil, "AddrInfo", esil_addrinfo, 1, 1, R_ANAL_ESIL_OP_TYPE_UNKNOWN);
```

Comment:
segfault in just a step
![IMAGE 2019-06-20 12:15:25](https://user-images.githubusercontent.com/917142/59841663-1710f000-9355-11e9-9d89-a27bdfe31a0c.jpg)